File: sum.py
import datetime
NUMBER = 50000000

def sum(x):
	_sum = 0
	for i in xrange(x+1):
		_sum += i
	return _sum

a = datetime.datetime.now()
result = sum(NUMBER)
b = datetime.datetime.now()
c = b - a

print "Result: ", result
print "Elapsed: %dms" % int(c.total_seconds()*1000)

File: sum.js
var NUMBER = 50000000;

function sum(x) {
  var sum = 0;
  for (var i = 0; i <= x; i++) {
    sum += i;
  }
  return sum;
}

var start = new Date().getTime();
var result = sum(NUMBER);
var end = new Date().getTime();

console.log("Result: ", result);
console.log("Elapsed: %dms", end-start);